redis/deprecated/redisbp/example_config_test.go

```diff
@@ -3,13 +3,11 @@ package redisbp_test
 import (
 	"context"
 
-	"github.com/go-redis/redis/v7"
 	"github.com/reddit/baseplate.go"
-	"github.com/reddit/baseplate.go/ecinterface"
 	"github.com/reddit/baseplate.go/redis/deprecated/redisbp"
 )
 
-type Config struct {
+type config struct {
```

Comment:
if you don't export this, godoc won't render the whole file example (it will only render the example function)